openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -59,7 +64,7 @@ class standard_deduction(Variable):
     def formula(tax_unit, period, parameters):
         # Calculate basic standard deduction
         basic_stded = tax_unit("basic_standard_deduction", period)
-        STD = parameters(period).tax.deductions.standard
+        CHARITY = parameters(period).tax.deductions.itemized.charity
```

Comment:
I understand capitalizing acronyms but this seems to veer into capitalization that should be reserved for constants

Reply:
Makes sense, I changed it to lowercase